### PR TITLE
update rolebindingaccessor to use generated clients

### PR DESCRIPTION
**Pull request by deads2k**
@openshift/sig-security fyi

### CODE REVIEW

**Comment by deads2k** [comment]
@mfojtik helps with test legacy client removal

File: test/util/server/server.go [hunk 1]
```diff
@@ -607,18 +608,24 @@ func WaitForServiceAccounts(clientset kclientset.Interface, namespace string, ac
 
 // CreateNewProject creates a new project using the clusterAdminClient, then gets a token for the adminUser and returns
 // back a client for the admin user
-func CreateNewProject(clusterAdminClient *client.Client, clientConfig restclient.Config, projectName, adminUser string) (kclientset.Interface, *client.Client, error) {
+func CreateNewProject(clusterAdminClient *client.Client, clientConfig restclient.Config, projectName, adminUser string) (kclientset.Interface, *client.Client, *restclient.Config, error) {
+	authorizationClient, err := authorizationclient.NewForConfig(&clientConfig)
```
Comment (deads2k) [inline test/util/server/server.go]: @mfojtik did we solve this two different ways?  this prevents a proliferation of different client return values from the various helpers.

File: pkg/oc/admin/project/new_project.go [hunk 2]
```diff
@@ -61,6 +63,9 @@ func NewCmdNewProject(name, fullName string, f *clientcmd.Factory, out io.Writer
 			if options.Client, _, err = f.Clients(); err != nil {
 				kcmdutil.CheckErr(err)
 			}
+			authorizationClient, err := f.OpenshiftInternalAuthorizationClient()
```
Comment (enj) [inline pkg/oc/admin/project/new_project.go]: Seems like most of the logic is this area should be moved into the `complete` func.

**Comment by deads2k** [comment]
> Seems like most of the logic is this area should be moved into the complete func.

Your point is taken.  I added this where the others were.  When we make the final sweep, I'll see how bad it is to move.

I'll be rebasing https://github.com/openshift/origin/pull/16550 on this.  I'd like it to merge overnight.

/retest

**Comment by kargakis** [comment]
flake was https://github.com/openshift/origin/issues/16592

/retest

**Comment by deads2k** [comment]
our code is using the wrong the API.  Pre-discussed this bug with @smarterclayton

**Comment by kargakis** [comment]
> @deads2k deads2k added the approved label 2 hours ago

I would suggest using the bot for adding labels. Especially, the approved label since it's handled by mungegithub and not prow.

**Pull request merged**